src/test/java/nextstep/subway/acceptance/SectionAcceptanceTest.java

```diff
@@ -0,0 +1,210 @@
+
+        //then
+        var line = LineCommonApi.findLineById(인천1호선.getId()).as(LineResponse.class);
+        assertThat(line.getStations().size()).isEqualTo(3);
+
+        var addedSection = response.as(SectionResponse.class);
+        assertThat(addedSection).isEqualTo(new SectionResponse(국제업무지구역.getId(), 송도달빛축제공원역.getId(), 3));
+    }
+
+    /**
+     * Given: 등록된 노선이 있고
+     * When: 관리자가 새로운 구간을 노선의 가운데에 추가하면
+     * Then: 노선에 새로운 구간이 추가된다.
+     */
+    @DisplayName("지하철 구간을 노선의 가운데에 등록한다.")
+    @Test
+    void addSectionToMiddle() {
+        //given
+        var 인천터미널역 = StationCommonApi.createStation("인천터미널역").as(StationResponse.class);
+        SectionCreateRequest request = new SectionCreateRequest(계양역.getId(), 인천터미널역.getId(), 5);
+
+        //when
+        var response = LineCommonApi.addSection(인천1호선.getId(), request);
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+
+        //then
+        var line = LineCommonApi.findLineById(인천1호선.getId()).as(LineResponse.class);
+        assertThat(line.getStations().size()).isEqualTo(3);
+
+        var addedSection = response.as(SectionResponse.class);
+        assertThat(addedSection).isEqualTo(new SectionResponse(계양역.getId(), 인천터미널역.getId(), 5));
+    }
+
+    /**
+     * Given: 등록된 노선이 있고
+     * When: 관리자가 새로운 구간을 노선의 처음에 추가하면
+     * Then: 노선에 새로운 구간이 추가된다.
+     */
+    @DisplayName("지하철 구간을 노선의 처음에 등록한다.")
+    @Test
+    void addSectionToFirst() {
+        //given
+        var 신검단중앙역 = StationCommonApi.createStation("신검단중앙역").as(StationResponse.class);
+        SectionCreateRequest request = new SectionCreateRequest(신검단중앙역.getId(), 계양역.getId(), 3);
+
+        //when
+        var response = LineCommonApi.addSection(인천1호선.getId(), request);
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+
+        //then
+        var line = LineCommonApi.findLineById(인천1호선.getId()).as(LineResponse.class);
+        assertThat(line.getStations().size()).isEqualTo(3);
+
+        var addedSection = response.as(SectionResponse.class);
+        assertThat(addedSection).isEqualTo(new SectionResponse(신검단중앙역.getId(), 계양역.getId(), 3));
+    }
+
+    /**
+     * Given: 등록된 노선과 노선에 추가로 등록된 구간이 있고
+     * When: 관리자가 새로운 구간을 삭제하면
+     * Then: 새로운 구간이 제거된다.
+     */
+    @DisplayName("지하철 구간을 삭제한다.")
+    @Test
+    void deleteSection() {
+        //given
+        SectionCreateRequest request = new SectionCreateRequest(국제업무지구역.getId(), 송도달빛축제공원역.getId(), 3);
+        LineCommonApi.addSection(인천1호선.getId(), request);
+
+        //when
+        LineCommonApi.deleteSection(인천1호선.getId(), 송도달빛축제공원역.getId());
+
+        //then
+        LineResponse line = LineCommonApi.findLineById(인천1호선.getId()).as(LineResponse.class);
+        List<String> names = line.getStations().stream().map(StationResponse::getName).collect(Collectors.toList());
+
+        assertThat(names).containsExactly("계양역", "국제업무지구역");
```

Comment:
값을 하드 코딩해서 검증하는 대신에, 위에서 정의한 `StationResponse`에서 값을 꺼내서 비교하는 건 어떨까요?
```suggestion
        assertThat(names).containsExactly(계양역.getName(), 국제업무지구역.getName());
```